### DISCUSSION TITLE
Support CEL for health checks

### DISCUSSION THREAD

**Discussion by stevehipwell** [projectsveltos/addon-controller#766] [Ideas]
Has there been any consideration given to supporting ~~[CUE](https://cuelang.org/)~~ [CEL](https://github.com/google/cel-go) for health checks?

**Comment by gianlucam76** [comment]
Hi @stevehipwell I always wanted to add CUE support. I have not had the time yet. If we find a way to pass either Lua or CUE without changing the rest of the flow, this would be a great addition for Sveltos.

Potentially this might also be used with event framework so to make it consistent to all Sveltos features. 

Would you mind filing an enhancement request with an example you had in mind? Thank you

**Reply by stevehipwell** [reply]
@gianlucam76 which repo would you like the enhancement request in? I also wasn't sure about linking healthcheck and event framework both using CUE vs individual?

**Comment by gianlucam76** [comment]
Thanks @stevehipwell 

We can use libsveltos repo.
